docs/markdown/Reference-manual.md

```diff
@@ -1502,11 +1506,19 @@ the following methods:
   an array containing only the arguments supported by the compiler,
   as if `has_argument` were called on them individually.
 
+- `get_supported_link_arguments(list_of_string)` *(added 0.46.0)* returns
+  an array containing only the arguments supported by the linker,
+  as if `has_link_argument` were called on them individually.
+
 - `has_argument(argument_name)` returns true if the compiler accepts
   the specified command line argument, that is, can compile code
-  without erroring out or printing a warning about an unknown flag,
-  you can specify external dependencies to use with `dependencies`
-  keyword argument.
+  without erroring out or printing a warning about an unknown flag.
```

Comment:
Mention of `dependencies`  is removed because this is a documentation error and this keyword doesn't really exist?

Reply:
Right, I don't know why the doc mention that, but the code doesn't accept any kwargs, that method is decorated with `@permittedMethodKwargs({})`.